### ISSUE TITLE
Change urls from the "develop" branch to pull from "Development" 

### ISSUE THREAD

**Issue by aziza-twdb** [TNRIS/tnris.org#486]
**UPDATE WE FIXED IT, DISREGARD**
Since we just made a fresh copy from the master branch to get back on board from our errors this morning, we are just going start from scratch as a whole. I would like the environment to see changes made to our “development” branch on this URL http://develop.tnris.org/ Currently it's pulling from the old branch that we had our errors on and we are going to eventually phase one that out.

**Expected** 

Development branch will be available for viewing under this link here http://development.tnris.org/

**Comment by space-longoria** [comment]
I'm not sure how develop.tnris.org is connected to the develop branch on the backend or what goes into getting the URL for development.tnris.org. If it's not feasible to move develop.tnris.org, could we merge development into develop and then use the same develop>master pipeline build?

Also, what is the technical explanation behind merging master to develop creating a +200 commit push? Is this something I should have been doing regularly as part of the pipeline process? If so, that's a big my bad and would want to incorporate that into Aziza's workflow moving forward.

**Comment by space-longoria** [comment]
Development has been merged into develop. I tested the excessive code push (+188) and it went through fine. I waited a bit and triple checked to make sure we're not losing visual/code changes on the site, everything looks good. I'm guessing the commits that were pushed straight to master were security updates/dependabot skipping the develop branch of the pipeline. **I would definitely suggest to merge master into develop weekly or monthly going forward** to prevent this from happening again. The development branch can be sunset.
